.github/workflows/pypi-publish.yml

```diff
@@ -20,10 +20,6 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: 3.9
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
```

Comment:
Is this being done elsewhere or just no longer necessary for uploading  to PyPI?

Reply:
The pip upgrading, I thought I would defer to github actions to have dealt with.

The other dependency installation is now in the Makefile command.